src/Microsoft.Data.SqlClient/tests/ManualTests/DataCommon/DataTestUtility.cs

```diff
@@ -73,6 +74,11 @@ public static class DataTestUtility
         private static Dictionary<string, bool> AvailableDatabases;
         private static BaseEventListener TraceListener;
 
+        //Kerberos variables
+        public static readonly string KerberosDomainUser = null;
+        internal static readonly string KerberosDomainPassword = null;
+        public static readonly bool IsKerberos = false;
```

Comment:
This public field and the public property below are redundant. We could encapsulate the logic in a single property like: 
`public static bool IsKerberos => !string.IsNullOrEmpty(KerberosDomainUser) && !string.IsNullOrEmpty(KerberosDomainPassword);`

Also, DataTestUtility.IsKerberos doesn't really make a lot of sense to me.